### ISSUE TITLE
bioconda::bowtie and bowtie2 not available on win-64

### ISSUE THREAD

**Issue by matsojr22** [ZadorLaboratory/mapseq-processing#2]
It seems that some of the dependencies bowtie and bowtie2 are not available for win-64 in the bioconda channel. Is this going to be a problem as a windows user?

**Comment by matsojr22** [comment]
I just recalled that John mentioned that this was not yet possible, just checking in on the status. Using a linux VM was our proposed current work around.

**Comment by jhover** [comment]
Yep, we have not targeted native Windows yet. Just Linux, MacOS Intel, MaxOS Apple Silicon, and WSL.
